### PR TITLE
bug with text.replaced fixed

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/80

### CODE REVIEW

**Comment by EugeneDar** [comment]
@levBagryansky please, have a look

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java [hunk 1]
```diff
@@ -54,8 +54,8 @@ public class EOtext$EOreplaced extends PhDefault {
      *
      * @param sigma Sigma
      * @todo #21:30min Current implementation using Java.
-     *  We should implement it only via EOLANG code. We can do
-     *  it by using reduce method
+     * We should implement it only via EOLANG code. We can do
```
Comment (levBagryansky) [inline src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java:57]: As I know `@todo` description must have a one more intendation than `@todo`. That's why there were spaces here.

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java [hunk 2]
```diff
@@ -66,12 +66,14 @@ public class EOtext$EOreplaced extends PhDefault {
...
                 rho -> {
-                    final String target = new Dataized(rho.attr("target").get())
-                        .take(String.class);
+                    final String target = (new Dataized(rho.attr("target").get())
+                        .take(String.class)).replaceAll("\\$", "\\\\\\$");
                     final String replacement = new Dataized(rho.attr("replacement").get())
                         .take(String.class);
                     final Phi text = rho.attr("ρ").get();
                     final String content = new Param(text, "s").strong(String.class);
+                    System.out.println(content + " " + target + " " + replacement + " -> " + content.replaceAll(target, replacement));
```
Comment (levBagryansky) [inline src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java:75]: Are this debug prints?

**Review by levBagryansky — CHANGES_REQUESTED** [review]

**Comment by levBagryansky** [comment]
@EugeneDar Why test were not started?

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java (hunk 2 above)
Comment (EugeneDar) [reply src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java:75]: @levBagryansky yes, I'll remove it.

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java [hunk 3]
```diff
@@ -66,8 +66,8 @@ public class EOtext$EOreplaced extends PhDefault {
             new AtComposite(
                 this,
                 rho -> {
-                    final String target = new Dataized(rho.attr("target").get())
-                        .take(String.class);
+                    final String target = (new Dataized(rho.attr("target").get())
+                        .take(String.class)).replaceAll("\\$", "\\\\\\$");
```
Comment (levBagryansky) [inline src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java:70]: @EugeneDar I think I would be good to leave `@todo` here with task to decide what to do with other special symbols and than do smth with them.

**Review by levBagryansky — CHANGES_REQUESTED** [review]

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java (hunk 3 above)
Comment (levBagryansky) [reply src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java:70]: @EugeneDar WDYT?

Comment (EugeneDar) [reply src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java]: @levBagryansky I fixed this

**Review by levBagryansky — APPROVED** [review]

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by EugeneDar** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**